docs/changelog.rst

```diff
@@ -6,6 +6,7 @@ Unreleased
 
 * Update pytket version requirement to 1.26.
 * Update qiskit-aer version requirement to 0.14.
+* Add `IBMQBackend.default_compilation_pass_static` for offline compilation given config and props objects.
```

Comment:
I would suggest to put `offline` in the name somewhere, maybe just do ``IBMQBackend.default_compilation_pass_offline` ?

Reply:
Sounds good. Done.